Bottom: 2d7d36989522b9e010497256793038fb2cb5aa9e
Top:    4f6b6265d543312f4e8c9d74b6522b4168986bb9
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-07-26 15:37:14 -0700

Refresh of mpi-finalize

---

diff --git a/src/preload.c b/src/preload.c
index 52eaf1a..996d8e0 100644
--- a/src/preload.c
+++ b/src/preload.c
@@ -798,9 +798,12 @@ int select(int nfds, fd_set *readfds, fd_set *writefds,
 	return ret;
 }
 
+#include <stdio.h>
+
 int shutdown(int socket, int how)
 {
 	int fd;
+	printf("shutdown how %s\n", how == SHUT_RDWR ? "RDWR" : how == SHUT_RD ? "RD" : "WR");
 	return (fd_get(socket, &fd) == fd_rsocket) ?
 		rshutdown(fd, how) : real.shutdown(fd, how);
 }
